File: fizzbuzz-test.php
<?php
for ($i = 1; $i <= 100; $i++) {
    $output = '';
    switch (true) {
        case ($i % 3 == 0 && $i % 5 == 0):
            $output = 'FizzBuzz';
            break;
        case ($i % 3 == 0):
            $output = 'Fizz';
            break;
        case ($i % 5 == 0):
            $output = 'Buzz';
            break;
        default:
            $output = $i;
    }
    echo $output . "\n";
}

File: fizzbuzz-test.go
package main

import (
    "fmt"
)

func main() {
    for i := 1; i <= 100; i++ {
        var output string
        switch {
        case i % 3 == 0 && i % 5 == 0:
            output = "FizzBuzz"
        case i % 3 == 0:
            output = "Fizz"
        case i % 5 == 0:
            output = "Buzz"
        default:
            output = fmt.Sprint(i)
        }
        fmt.Println(output)
    }
}
